3_javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,68 @@
+
+const ApiKey = `https://open.er-api.com/v6/latest/EUR`;
+
+
+
+let toSelectList = document.createElement("select");
```

Comment:
Here you could use a const instead of let. Can you guess why?

Reply:
yes because it is global variable 😀.